# HG changeset patch
# Parent 9e5e6192c320b1ea461ee8c132ae376b4c2a6e71
Get libxl to use the format "tapx.y" for emulated NICs, rather than "vifx.y-emu"

This is needed for backwards compat with qemu-dm, which seems to expect this
(for now at least).

diff -r 9e5e6192c320 tools/libxl/libxl.c
--- a/tools/libxl/libxl.c
+++ b/tools/libxl/libxl.c
@@ -3126,7 +3126,7 @@
     case LIBXL_NIC_TYPE_VIF:
         return GCSPRINTF("vif%u.%d", domid, devid);
     case LIBXL_NIC_TYPE_VIF_IOEMU:
-        return GCSPRINTF("vif%u.%d" TAP_DEVICE_SUFFIX, domid, devid);
+        return GCSPRINTF("tap%u.%d", domid, devid);
     default:
         abort();
     }
